src/main/java/com/trolley/trolley/Payment.java

```diff
@@ -292,18 +307,138 @@ public Batch getBatch() {
         return batch;
     }
 
-    public void setBatch(Batch batch) {
+    public void setBatch(final Batch batch) {
         this.batch = batch;
     }
 
     public String getCheckNumber() {
         return checkNumber;
     }
 
-    public void setCheckNumber(String checkNumber) {
+    public void setCheckNumber(final String checkNumber) {
```

Comment:
Wasn't sure what the convention was with adding `final` - noticed most had it, except these two.